src/foam/nanos/cron/Schedulable.js

```diff
@@ -161,7 +161,7 @@ foam.CLASS({
         return d != null ? d : null;
       `,
       factory: function(){
-        return `${this.objectToSchedule.toSummary()} - ${this.nextScheduledDate}`;
+        return `${this.objectToSchedule?.toSummary()} ${this.objectToSchedule?.toSummary() && this.nextScheduleDate ? '-' : ''} ${this.nextScheduledDate}`;
```

Comment:
How about we simplify to the below, which is easier to read I think:
return this.objectToSchedule ?
  `${this.objectToSchedule.toSummary()} - ${this.nextScheduledDate}` :
  this.nextScheduledDate;

Reply:
@kgrgreer if `this.objectToSchedule ` is not null and `this.nextScheduleDate` is undefined, in that case, also we don't need '-'?
In that case, I think we also need to check if `this.nextScheduledDate` is undefined or not and we shouldn't add this('-').

Reply:
I agree if nextScheduledDate is ever null. Is it?

Reply:
yes, its initial value is undefined.

Reply:
K wait I think this is a mistake. This factory should be for Schedulable.name, not for Schedulable.nextScheduledDate

Reply:
Yes, why is the a js factory on this property any way?

Reply:
This was my mistake, think I was resolving a merge conflict or something and it ended up here.